app/helpers/field_validation_helper.rb

```diff
@@ -74,6 +74,17 @@ def validate_checkbox_field(page, values:, allowed_values:, other: false, other_
     []
   end
 
+  def validate_qualification_or_not(page, values:)
```

Comment:
Since this only performs validation for a single form field and is specific to a question, I think it'd be better to have this method in the controller itself.  I can't see where it would be reused, and all the other methods in this helper are quite generic.

Reply:
👍